MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -2942,11 +2942,16 @@ public void resetPilotAndEntity() {
             //This double enumeration is annoying to work with for crew-served units.
             //Get the option names while we enumerate so they can be used later
             List<String> optionNames = new ArrayList<String>();
+            List<String> cyberOptionNames = new ArrayList<String>();
```

Comment:
Make this `Set<String> cyberOptionNames = new HashSet<String>();`. Running `List<T>::contains` is a much more expensive operation than `HashSet<T>::contains`.

Reply:
Fixed.